spec/lib/metasploit/framework/login_scanner/postgres_spec.rb

```diff
@@ -40,7 +41,7 @@
 
     context 'when there is no realm on the credential' do
       it 'uses template1 as the default realm' do
-        expect(Msf::Db::PostgresPR::Connection).to receive(:new).with('template1', 'root', 'toor', 'tcp://:', nil)
+        expect(Msf::Db::PostgresPR::Connection).to receive(:new).with('template1', 'root', 'toor', 'tcp://127.0.0.1:5432', nil, nil)
```

Comment:
Due to `set_sane_defaults` previously having been declared outside the scanner class in `lib/metasploit/framework/login_scanner/postgres.rb` the port default was never being set for the scanner. So I have updated the test and also added a host to improve the readability of the test output.